prov/efa/src/rxr/rxr_pkt_entry.h

```diff
@@ -175,21 +175,23 @@ struct rxr_pkt_rx_key {
 	fi_addr_t addr;
 };
 
+struct rxr_op_entry;
```

Comment:
Why is this declaration necessary?

Reply:
Note that `rxr_op_entry.h` include `rxr_pkt_entry.h`

but `rxr_pkt_entry.h`still reference `rxr_op_entry` through some pointer types. Therefore this forward declaration is needed.